src/tsickle.ts

```diff
@@ -106,6 +106,16 @@ function hasModifierFlag(node: ts.Node, flag: ts.ModifierFlags): boolean {
   return (ts.getCombinedModifierFlags(node) & flag) !== 0;
 }
 
```

Comment:
Done. In a nutshell, TypeTranslator has a map of import aliases, symbol->string. If we resolve aliases before passing them to TypeTranslator, its local alias map is defeated as it only contains the original, non-dereferenced symbol.